tests/jobs/localisation/test_configs/test_field_config.py

```diff
@@ -174,11 +174,8 @@ def test_correlation_config_ref_point(field_config, surface_config):
         ),
     ],
 )
-def test_correlation_config_no_ref_point(field_config, surface_config):
-    # Create a surface file (should only exist)
-    with open("surface_file.txt", "w", encoding="utf-8") as fout:
-        fout.write("A test file")
-
+def test_correlation_config_no_ref_point(field_config, surface_config, monkeypatch):
+    monkeypatch.setattr(os.path, "exists", MagicMock())
```

Comment:
The reason `monkeypatch` does not work is because you are not patching the correct thing. You refactored it from using `os.path.exists` to `pathlib`, so what would work is:
```
monkeypatch.setattr(pathlib.Path, "exists", MagicMock())
```
(also need to import pathlib)

Reply:
OK